data/hai/hai ships.txt

```diff
@@ -1553,7 +1553,7 @@ ship "Solifuge"
 	attributes
 		category "Heavy Warship"
 		licenses
-			"Unfettered Militia"
+			"True Unfettered License"
```

Comment:
```suggestion
			"True Unfettered"
```
I believe that "License" is added to the names of license outfits automatically, and hence is not needed here. In the same way as navy ships say `licenses "Navy"` rather than `licenses "Navy License"`.